DXRModules/DeusEx/Classes/DXRFixup.uc

```diff
@@ -834,3 +834,23 @@ function SetAllLampsState(bool type1, bool type2, bool type3, optional Vector lo
     }
 #endif
 }
+
+function MoveSmugglerElevator()
+{
+    local #var(prefix)DeusExMover elevator;
+    local bool street;
+
+    if (!#defined(vanilla)) return;
```

Comment:
Probably just put this check into PreBeginPlay and Destroy() it if it shouldn't exist